build/assets/scripts/high-performance-hooks.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pid=$(jq '.pid' /dev/stdin)
+mask=$1
+ns=$(ip netns identify "${pid}")
+
+[[ -n "${pid}" ]] && [[ -n "${mask}" ]] && [[ -n "${ns}" ]] || exit 0
+
+ip netns exec "${ns}" mount -o remount,rw /sys
+ip netns exec "${ns}" find /sys/devices -type f -name rps_cpus -exec sh -c "echo ${mask} | cat > {}" \;
+ip netns exec "${ns}" mount -o remount,ro /sys
```

Comment:
I wonder if it makes sense to split this script in two, one which identifies the NS and executes the real script in the right namespace, and another one which just does the work. Something like

```
$ cat assets/scripts/run-in-ns.sh
#!/usr/bin/env bash

pid=$(jq '.pid' /dev/stdin)
mask=$1
what=$2
ns=$(ip netns identify "${pid}")

[[ -n "${pid}" ]] && [[ -n "${mask}" ]] && [[ -n "${ns}" ]] || exit 0

ip netns exec "${ns}" ${what}
```
and
```
$ cat assets/scripts/high-performace-hooks.sh
#!/usr/bin/env bash
mount -o remount,rw /sys
find /sys/devices -type f -name rps_cpus -exec sh -c "echo ${mask} | cat > {}" \;
mount -o remount,ro /sys
```
Whatever's easier to write and maintain, I guess.